media/src/androidMain/kotlin/dev/icerock/moko/media/BitmapUtils.kt

```diff
@@ -49,5 +51,44 @@ object BitmapUtils {
         return Bitmap.createScaledBitmap(bitmap, width, height, true)
     }
 
+    fun getBitmapForStream(
+        inputStream: InputStream,
+        maxImageWidth: Int,
+        maxImageHeight: Int
+    ): Bitmap {
+        val bitmapOptions = BitmapFactory.Options().apply {
+            inJustDecodeBounds = true
+        }
+        val bitmapBytes = inputStream.readBytes()
+        BitmapFactory.decodeByteArray(bitmapBytes, 0, bitmapBytes.size, bitmapOptions)
```

Comment:
decodeStream is better https://developer.android.com/reference/android/graphics/BitmapFactory#decodeStream(java.io.InputStream,%20android.graphics.Rect,%20android.graphics.BitmapFactory.Options)

Reply:
`decodeByteArray` to `decodeStream` was replaced.